sdk/keyvault/azure-keyvault-keys/tests/test_challenge_auth.py

```diff
@@ -69,6 +69,32 @@ def test_multitenant_authentication(self, client, is_hsm, **kwargs):
         else:
             os.environ.pop("AZURE_TENANT_ID")
 
+    @pytest.mark.skip("Manual test for specific, CAE-enabled environments.")
+    @pytest.mark.live_test_only
+    def test_cae_live(self, **kwargs):
```

Comment:
I personally wouldn't commit this because it's useful only given a vault that always responds with CAE challenges, which we won't always have, and it doesn't add coverage we can't get with mocks